Open mSupply central (#28)
msupply-foundation/msupply_docs: content/docs/introduction/requirements.md

| 1.1.00 - 1.1.16  | 7.04.01+              |
| 1.2.00+          | 7.05.05+              |
| 1.4.00+          | 7.09.00+              |
| 2.0.00+          | 7.14.00+              |

If you attempt to connect to an incompatible server you'll get an error message like this:

![server version mismatch!](/docs/introduction/images/version_mismatch.png)

> ## Open mSupply Central Server

Reviewer: To be in line with other usages, and the fact that `central` and `server` aren't proper nouns in themselves, and nor is `central server` in this case
```suggestion
## Open mSupply central server
```

> From v2.0.00, open mSupply central server is required for succesfull synchronisation. See open mSupply central server section and open [mSupply central server site configuration in mSupply](https://docs.msupply.org.nz/synchronisation:sync_sites#open_msupply_central_server_settings) for more details. 

Reviewer 2: ```suggestion
From v2.0.00, the Open mSupply central server is required for successful synchronisation. See the [Open mSupply central server](/docs/introduction/central) and [Open mSupply central server site configuration in mSupply](https://docs.msupply.org.nz/synchronisation:sync_sites#open_msupply_central_server_settings) for more details. 
```

> You will get the following error message if open mSupply central server is not configured

Reviewer 2: ```suggestion
You will get the following error message the Open mSupply central server is not configured
```
